azurelinuxagent/common/protocol/wire.py

```diff
@@ -1200,6 +1302,16 @@ def get_artifacts_profile(self):
                                  message=msg,
                                  log_event=False)
 
+                try:
+                    file_name = ARTIFACTS_PROFILE_BLOB_FILE_NAME.format(artifacts_profile.get_sequence_number())
+                    file_path = os.path.join(conf.get_lib_dir(), file_name)
+                    self.save_cache(file_path, profile)
+                except Exception:
+                    logger.warn("Could not save artifacts profile blob to disk")
```

Comment:
Consider adding more information about the exception? `save_cache` doesn't log anything, it just raises and here that message is lost.

Reply:
added the exception message